Dockerfile.local

```diff
@@ -46,9 +46,9 @@ RUN --mount=type=cache,id=gomod,target=/go/pkg/mod \
     ./scripts/download
 
 COPY ./cmd ./cmd
-COPY ./pkg ./pkg
 COPY ./tests ./tests
 COPY ./.git ./.git
+COPY ./pkg ./pkg
```

Comment:
This is a docker caching optimization, as the `COPY ./tests ./tests` line will now be cached most of the time.